### ISSUE TITLE
Focus indicator contrast on audio player too low (WCAG 1.4.11) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#223]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
**Page:**
Course Overview page
Lesson 1: Overview of electronic Passport Next Generation
Lesson 2: Domestic Print
Lesson 3: Observation Print
Lesson 4: Manual QA
Lesson 5: Reports

**Description:**
The focus indicator in the audio player has insufficient color contrast. 

The element was calculated to have the following traits
   • Foreground: rgb(185, 17, 113)
   • Background: rgb(76, 76, 76)
   • The contrast ratio is: 1.4:1

Any visual information necessary to indicate a state, such as whether a component is selected or focused must also ensure that the information used to identify the control in that state has a minimum 3:1 contrast ratio.
![image](https://user-images.githubusercontent.com/93537804/231588015-c5469e0c-edc1-4188-8e37-ff2b9bff8e68.png)
*******************************************************************************
